src/coreclr/src/jit/gentree.cpp

```diff
@@ -5398,6 +5398,13 @@ bool GenTree::OperIsImplicitIndir() const
         case GT_ARR_ELEM:
         case GT_ARR_OFFSET:
             return true;
+#ifdef FEATURE_SIMD
+        case GT_SIMD:
+        {
+            GenTreeSIMD* simdNode = (const_cast<GenTree*>(this))->AsSIMD();
+            return simdNode->OperIsMemoryLoad();
+        }
+#endif // FEATURE_SIMD
 #ifdef FEATURE_HW_INTRINSICS
         case GT_HWINTRINSIC:
```

Comment:
Can you please make `GenTreeHWIntrinsic::OperIsMemoryLoadOrStore()` method `const` and remove the cast?

Reply:
OK